test/test_tidy.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "helper"
```

Comment:
None in particular.
Just that how `test_tidy` is closer to the imperative form of `testing_tidy`..